src/validateCase.js

```diff
@@ -0,0 +1,7 @@
+function validateCase(caseName) {
+  return ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(caseName);
```

Comment:
Create a constant for `['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER']` and use it to generate an error message with available cases